Sources/Components/Control/ReviewButtonControl/ReviewButtonControl.swift

```diff
@@ -0,0 +1,93 @@
+//
+//  Copyright © 2019 FINN AS. All rights reserved.
+//
+
+import UIKit
+
+public protocol ReviewButtonControlDelegate: AnyObject {
+    func reviewButtonControl(_ reviewButtonControl: ReviewButtonControl, giveReviewWasTapped startReview: Bool)
+}
+
+public final class ReviewButtonControl: UIControl {
+    // MARK: - Public
+
+    public weak var delegate: ReviewButtonControlDelegate?
+    public var text: String = "" {
+        didSet {
+            titleLabel.text = text
+        }
+    }
+    public static let cornerRadius: CGFloat = 8
```

Comment:
Doesn't really need to be neither public nor static, right? 
(Probably doesn't need to be a  property either, since it seems to be used once, and only in the setup).